app/assets/javascripts/notifications.js

```diff
@@ -109,6 +114,17 @@ function quickCreate(form, data_type) {
   });
 }
 
+function closeQuickCreate() {
+  if ($(this).closest('.quick_create').attr('id') === 'category_quick_create' && !$('#category_quick_create').hasClass('display_none')) {
```

Comment:
This function looks good, but there looks like we have a lot of repetitive logic here the only difference is the id of the closet `.quick_create` attribute. 

We could do something like this:
```
function closeQuickCreate() {
  var quickCreateId = $(this).closest(".quick_create").attr("id");
  if ((quickCreateId === "category_quick_create" || quickCreateId === "mood_quick_create" || quickCreateId === "strategy_quick_create") && quickCreateId.hasClass("display_none")) {
  quickCreateId.toggleClass("display_none");
}
  hideBackdrop();
}
```